relay-filter/src/csp.rs

```diff
@@ -47,7 +47,7 @@ pub fn should_filter(event: &Event, config: &CspFilterConfig) -> Result<(), Filt
 
 /// A pattern used to match allowed paths.
 ///
-/// Scheme, domain and port are extracted from an url,
+/// Scheme, domain, port and the remaining path are extracted from an URL,
```

Comment:
Not sure if I should modify the name of the struct, I think it's not necessary but open to other ideas.

Reply:
So I would rename this. It becomes a `URI` actually at this point, and maybe that's the name I would use. Or something like `PatternUri` 🤷